grails-app/assets/javascripts/occurrenceMap.js

```diff
@@ -390,8 +390,25 @@ OccurrenceMap.prototype.addGridLegendItem = function() {
         );
 }
 
+/*
+ * Helper for addLegendItem as that recieves its label as a dynamic message string
+ * and those messages aren't always translated
+ */
+function getLegendLabel(name) {
+    if(name in $.i18n.map) {
+        return $.i18n.prop(name);
+    }
+    var label = name.split('.');
+    label = label[label.length - 1];
+    label = label.match(/[A-Za-z][a-z,]*/g);
+
+    return label.map(function (word) {
+        return word.charAt(0).toUpperCase() + word.substring(1);
+    }).join(" ");
```

Comment:
``" -> '``. Also, redefining ``label`` is meh, especially since the type changes